third_party/move/move-examples/diem-framework/crates/crypto/Cargo.toml

```diff
@@ -38,7 +38,7 @@ byteorder = { workspace = true }
 criterion = { workspace = true }
 proptest = { workspace = true }
 proptest-derive = { workspace = true }
-ripemd160 = { workspace = true }
+ripemd = { workspace = true }
```

Comment:
There is a complaint about it from rust_fmt.sh.  I don't think it's currently being built.

Reply:
But this is not a formatting change, but change in the name of the package?

Reply:
The package is deprecated, and `scripts/rust_fmt.sh` runs `cargo machete` which checks and complains about it.  Since it's not actually built anyway it is harmless to fix it here.